### ISSUE TITLE
Sync of ontologies in Co web site  with Agroportal

### ISSUE THREAD

**Issue by elizabetharnaud** [bioversity/Crop-Ontology#193]
the sync has been suspended by Agroportal since few months waiting to find a solution about the way files are created always like if they were new ones. It creates a load on the job. Needs to be solved.

**Comment by jonquet** [comment]
More details on AgroPortal side here : 
https://github.com/agroportal/project-management/issues/167

basically we are waiting for either a fix of the "new file being generated" issue on cropontology.org website or a decision to switch to push mode on the cropontology.org side (rather than pull mode on the AgroPortal side). 

I am expressing concerns as the Crop Ontologies : 
http://agroportal.lirmm.fr/ontologies?filter=CROP 
Are not updated since January 2022.

**Comment by elizabetharnaud** [comment]
Is this issue solved?

**Comment by jonquet** [comment]
No.And the CO versions in AgroPortal are getting older and older without being archived.

**Comment by marieALaporte** [comment]
@jonquet is this fixed? I see that blueberry for instance has been loaded on AgroPortal

**Comment by jonquet** [comment]
No. And the problem is getting worst.
Still waiting for some changes on Cropontology.org web site. We can't fix this on AgroPortal side. 
The 2 options discussed in March 2022 (https://github.com/agroportal/project-management/issues/167) are either pull or push but both depend on Cropontology.org.

Blueberry has been uploaded manually by our team.
